### PR TITLE
fix(regtests): Bind to 127.0.0.1 and not localhost

### PR DESCRIPTION
**Pull request by royjacobson**
Apparently GitHub runners don't like IPv6?

### CODE REVIEW

File: tests/dragonfly/proxy.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ async def start(self):
 
     def drop_connection(self):
         """
-        Randomally drop one connection
+        Randomly drop one connection
```
Comment (adiholden) [inline tests/dragonfly/proxy.py:63]: in start server please use try except so we can see more clearly the errors

File: tests/dragonfly/replication_test.py [hunk 2]
```diff
@@ -1566,27 +1566,28 @@ async def test_network_disconnect(df_local_factory, df_seeder_factory):
     async with replica.client() as c_replica:
         await seeder.run(target_deviation=0.1)
 
-        proxy = Proxy("localhost", 1111, "localhost", master.port)
+        proxy = Proxy("127.0.0.1", 1111, "127.0.0.1", master.port)
         task = asyncio.create_task(proxy.start())
+        try:
+            await c_replica.execute_command(f"REPLICAOF localhost {proxy.port}")
```
[the remark this answers is not in the document]
Comment (royjacobson) [reply tests/dragonfly/replication_test.py:1573]: split start into start/serve.

File: tests/dragonfly/proxy.py (hunk 1 above)
Comment (royjacobson) [reply tests/dragonfly/proxy.py:64]: I don't follow.. How would that make the errors more clear?

Comment (adiholden) [reply tests/dragonfly/proxy.py:64]: print the exception

Comment (royjacobson) [reply tests/dragonfly/proxy.py:64]: f2f

**Review by adiholden — APPROVED** [review]

**Pull request merged**